protobuf/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.2.0-wip
```

Comment:
This could be a patch bump (we're not introducing new API), but a minor is fine as well.